### PR TITLE
Fixes 2 sloppy phrases

### PR DESCRIPTION
**Pull request by lamagnifica**
@stroilova please review

### CODE REVIEW

File: docs/en_us/dashboard/source/Reference.rst [hunk 1]
```diff
@@ -431,15 +431,15 @@ Submission Counts chart and report. The .csv file contains the following columns
        for these problems.
 
    * - ``consolidated_variant``
-     - TRUE if the problem uses the randomization setting in Studio, but there
-       is a problem with the Python script that randomizes the values. FALSE
-       if the Python script is randomizing values or if the problem does not
-       use the randomization setting.
+     - TRUE if the problem component uses the randomization setting in Studio,
+       but there is an error in the Python script that randomizes the values.
```
Comment (stroilova) [inline docs/en_us/dashboard/source/Reference.rst]: @brianhw , does this mean we only output consolidated_variant = TRUE if there is an error with the python script? For all well-formed problems, would this setting be FALSE? FYI: @lamagnifica  if I have misunderstood something.

Comment (brianhw) [inline docs/en_us/dashboard/source/Reference.rst]: Hmm.  I know nothing about a python "script" that randomizes the values, and what kind of error this is addressing.  My understanding was that this consolidation addressed the situation where a problem was not using the randomization setting, so answers were needlessly being labeled by their variants.  These are well-formed problems, but for which randomization is enabled but not used, and so its variants are consolidated and the flag set to TRUE.

Comment (lamagnifica) [inline docs/en_us/dashboard/source/Reference.rst]: @stroilova, how do you want to proceed?

Comment (lamagnifica) [inline docs/en_us/dashboard/source/Reference.rst]: @stroilova and @brianhw , how about this slightly better informed rewrite?
TRUE if the Studio Randomization setting for this problem component is set to Always, On Reset, or Per Student,  but there is no variation in the possible answers. Often, this indicates that the Python script that randomizes values for the problem is not present.
FALSE if the Studio Randomization setting for this problem component is set to Never or if the Python script is randomizing values.

Comment (brianhw) [inline docs/en_us/dashboard/source/Reference.rst]: Yes, thank you.  That seems much clearer.

**Comment by lamagnifica** [comment]
@stroilova and @brianhw I have reworded the definitions for the consolidated_variant column.

**Pull request merged**